Source/WebKit/WebProcess/WebProcess.cpp

```diff
@@ -447,7 +447,7 @@ void WebProcess::initializeWebProcess(WebProcessCreationParameters&& parameters)
             if (m_pagesInWindows.isEmpty() && critical == Critical::No)
                 critical = Critical::Yes;
 
-#if PLATFORM(MAC)
+#if PLATFORM(MAC) || PLATFORM(IOS_FAMILY)
```

Comment:
If this is Mac + all iOS things, maybe it should just be `#if PLATFORM(COCOA)`?

Reply:
Ah, good catch, will fix!

Thanks for reviewing!